Source/UIScrollView+EmptyDataSet.m

```diff
@@ -448,6 +448,9 @@ - (void)dzn_reloadEmptyDataSet
         
         DZNEmptyDataSetView *view = self.emptyDataSetView;
         
+        // Configure empty dataset fade in display
+        view.fadeInOnDisplay = [self dzn_shouldFadeIn];
```

Comment:
This line was moved up a little because in some cases, the method `didMoveToSuperview` was called before the value `fadeInOnDisplay` was set up. So I thought it would be better to set it before addSubview.